openapi-validator/src/main/java/io/ballerina/openapi/validator/ServiceValidator.java

```diff
@@ -324,12 +326,15 @@ private static void openAPIPathAgainstToBallerinaServicePath(DiagnosticSeverity
             }
         }
     }
+    public static String normalizedPath(String path) {
```

Comment:
Do we have a util class (e.g. `PathUtils`/`FileUtils`) to move this method?

Reply:
Currently, we haven't common util class there, will create and move this into it :)